AutomaticInterface/AutomaticInterface/AutomaticInterface.csproj

```diff
@@ -25,7 +25,7 @@
         <PackageLicenseExpression>MIT</PackageLicenseExpression>
         <EnableNETAnalyzers>True</EnableNETAnalyzers>
         <AnalysisLevel>latest-Recommended</AnalysisLevel>
-        <Version>1.5.0</Version>
+        <Version>1.5.1</Version>
```

Comment:
You are adding net7 as a target, can you bump the minor version instead of bugfix?

Reply:
Done